docs/how_to_create_a_module.md

```diff
@@ -4,164 +4,162 @@
-        # Available types: :integer, :boolean
-        # settings.attribute :vote_limit, type: :integer, default: 0
-      end
+     feature.settings(:global) do |settings|
+       # Add your global settings
+       # Available types: :integer, :boolean
+       # settings.attribute :vote_limit, type: :integer, default: 0
+     end
 
-      feature.settings(:step) do |settings|
-        # Add your settings per step
-      end
+     feature.settings(:step) do |settings|
+       # Add your settings per step
+     end
 
-      # # Register an optional resource that can be referenced from other resources.
-      # feature.register_resource do |resource|
-      #   resource.model_class_name = "Decidim::<EngineName>::<ResourceName>"
-      #   resource.template = "decidim/<engine_name>/<resource_view_folder>/linked_<resource_name_plural>"
-      # end
+     # # Register an optional resource that can be referenced from other resources.
+     # feature.register_resource do |resource|
+     #   resource.model_class_name = "Decidim::<EngineName>::<ResourceName>"
+     #   resource.template = "decidim/<engine_name>/<resource_view_folder>/linked_<resource_name_plural>"
+     # end
 
-      feature.register_stat :some_stat do |features, start_at, end_at|
-        # Register some stat number to the application
-      end
+     feature.register_stat :some_stat do |features, start_at, end_at|
+       # Register some stat number to the application
+     end
 
-      feature.seeds do |participatory_space|
-        # Define seeds for a specific participatory_space object
-      end
-    end
-    ```
+     feature.seeds do |participatory_space|
+       # Define seeds for a specific participatory_space object
+     end
+   end
+   ```
 
 1. Replace `Rakefile` with:
 
-    ```ruby
-    # frozen_string_literal: true
+   ```ruby
+   # frozen_string_literal: true
 
-    require "decidim/common_rake"
-    ```
+   require "decidim/common_rake"
+   ```
 
 1. Remove `MIT-LICENSE` and change `README`.
 
 1. Add `spec/spec_helper.rb` with:
 
-    ```ruby
-    # frozen_string_literal: true
+   ```ruby
+   # frozen_string_literal: true
 
-    require "decidim/dev"
+   require "decidim/dev"
 
-    ENV["ENGINE_NAME"] = File.dirname(__dir__).split("/").last
+   ENV["ENGINE_NAME"] = File.dirname(__dir__).split("/").last
 
-    Decidim::Dev.dummy_app_path = File.expand_path(File.join("..", "spec", "decidim_dummy_app"))
+   Decidim::Dev.dummy_app_path = File.expand_path(File.join("..", "spec", "decidim_dummy_app"))
 
-    require "decidim/dev/test/base_spec_helper"
-
-    ```
-
-1. Upload to GitHub with the naming *decidim-module-<engine_name>*, so it's easier to find on
-the [dependency graph](https://github.com/decidim/decidim/network/dependents).
+   require "decidim/dev/test/base_spec_helper"
+   ```
 
+1. Upload to GitHub with the naming _decidim-module-<engine_name>_, so it's easier to find on
+   the [dependency graph](https://github.com/decidim/decidim/network/dependents).
```

Comment:
Are all the indentation changes intentional?

Reply:
OMG, no. I'll take a stab at it tomorrow!